### ISSUE TITLE
Crash when creating new world

### ISSUE THREAD

**Issue by renadi** [dannydjdk/Nutritional-Balance#8]
I literally can't create a world with this mod installed on the latest Forge(1.16.5-36.2.2).  I fill out details, select create world and it CTD with the following error.  Also if I try to type a name when creating a world it CTD with the second error.  

Error 1(world creation) - https://pastebin.com/G45gX69n

Error 2(typing anything) - https://pastebin.com/dr8Y7J12

**Comment by dannydjdk** [comment]
Thanks for reporting this. I wasn't able to pin down the exact cause of this, but it appears to be related to forge's mapping changes, and the recent 1.16.5-0.7.6 release seems to fix it.

Strangely, I was not able to reproduce this on any existing modpacks or within the development environment no matter what forge version I used. But, I was able to reproduce it in a custom pack I made with MultiMC with just this mod and JEI. After updating to the official mapping, the problem resolved.

Let me know if 1.16.5-0.7.6 fixes the issue for you as well.
